content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -899,6 +899,61 @@ enum Role {
 </tab>
 </TabbedContent>
 
+## Defining composite types
+
+<Admonition type="info">
+
+Composite types are available in Preview in versions `3.10.0` and later.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Composite types are currently only available on MongoDB.
+
+</Admonition>
+
+Composite types (known as [embedded documents](https://docs.mongodb.com/manual/core/data-model-design/#std-label-data-modeling-embedding) in MongoDB) provide support for embedding records inside other records, by allowing you to define new object types. Composite types are structured and typed in a similar way to models.
+
+To use composite types with MongoDB, you will need to enable the `mongoDb` Preview feature in your `schema.prisma` file:
+
+```prisma file="schema.prisma"
+datasource db {
+  provider = "mongodb"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["mongoDb"]
+}
+```
+
+To define a composite type, use the `type` block. As an example, take the following schema:
+
+```prisma file=schema.prisma
+model Product {
+id     String  @id
+name   String
+photos Photo[]
+}
+
+type Photo {
+height Int
+width  Int
+url    String
+}
+```
+
+In this case, the `Product` model has an embedded `Photo[]` type.
```

Comment:
The `[]` is not really part of the type, but defines that it is a list of `Photo` types that can be stored in `photos`. Might be worth mentioning here.